FirebasePerformance.podspec

```diff
@@ -28,7 +28,7 @@ Firebase Performance library to measure performance of Mobile and Web Apps.
 
   base_dir = "FirebasePerformance/"
   s.source_files = [
-    base_dir + 'Sources/**/*.[mh]',
+    base_dir + 'Sources/**/*.[cmh]',
     base_dir + 'ProtoSupport/**/*.[mh]',
```

Comment:
Can the Protobuf dependency be removed below?

Reply:
We need to remove that. But that will be done after all the Protobuf dependencies are removed. For now, we will retain them until the last PR for the NanoPb conversion gets submitted.

Reply:
Good catch! I will remove this and all Protobuf files in another PR.